### ISSUE TITLE
Help wanted

### ISSUE THREAD

**Issue by sanni85** [LMFDB/lmfdb#1092]
I just finished uploading almost all lattice data computed by Anna (total 39,292 lattices). Only one lattice is missing the Leech lattice. When I try to run the import script on it I get the following message:

> DocumentTooLarge: command document too large

Here is the file [Leech.txt](https://github.com/LMFDB/lmfdb/files/239043/Leech.txt), it contains only one line, and its size is 5.7 Mb
Do you know how to import it?

To run the import script, I usually run sage in the folder lmfdb, and in sage I use the commands:
```
%run lmfdb/lattice/isom.py
%run lmfdb/import_scripts/lattices/lattice.py /path_to_file
```

**Comment by JohnCremona** [comment]
It seems thet this object is large enough to require gridFS.  Even though your text input file is <6MB and the mongo/bson limit is 16mb, it may be true that after creating the bson object to insert into the db it is > 16mb.

**Comment by fredstro** [comment]
Since I suffered similar problems in the past I took a look at this and it is indeed the bson encoding of integers which takes up a lot of space. I think the default encoding is set to 64bit integers.
If you don't want to use gridfs there are essentially two options:
1) store the data as a string instead of encoding the integers (I checked that for the Leech lattice this will give a db record of about 7MB). 
2) write your own integer codec class for bson

**Comment by sanni85** [comment]
Thank you Fredrik !
I will try to implement the first suggestion, convert the list to a string, it may solve the problem for now.

**Comment by sanni85** [comment]
This is fixed in pull request #1115

**Comment by jvoight** [comment]
Can I close this now?
